pkg/monitoring/poller.go

```diff
@@ -63,33 +66,34 @@ func (s *sourcePoller) Run(ctx context.Context) {
 		}
 	}
 
-	reusedTimer := time.NewTimer(s.pollInterval)
+	ticker := services.TickerConfig{
+		Initial:   utils.WithJitter(s.pollInterval),
+		JitterPct: 0.2,
```

Comment:
I lack knowledge on our chainlink-commons utils, but this looks like it has 2 jitter sources, is that so? Looking at https://github.com/smartcontractkit/chainlink-common/blob/ce5d667907ce901f088b1ac04fb06ea6295b8c9f/pkg/utils/utils.go#L15, the line 70 seems to be using a deprecated method that also introduces a 10% jitter. I like the 20% that you're using on line 71 better than the fixed, small & deprecated 10% of line 70 😄